### PR TITLE
Feat/exercises

### PR DESCRIPTION
**Pull request by MKuzich**
Cards styled completely, API changed to Service, Service adapted.

### CODE REVIEW

File: src/css/exercises.css [hunk 1]
```diff
@@ -120,28 +122,61 @@
...
   height: 18px;
   width: 18px;
-  padding: 2px;
+  padding: 1px;
+  flex-shrink: 0;
 }
 
-.exercises-favorite-btn svg {
+.exercises-delete-btn svg {
+  stroke: #242424;
```
Comment (snegurova) [inline src/css/exercises.css:156]: Could be variable

Comment (snegurova) [reply src/css/exercises.css:156]: The same in the whole file

File: src/js/api/ExercisesService.js [hunk 2]
```diff
@@ -77,31 +49,17 @@ export default class ExercisesService {
...
-   * @throws {Error} Throws an error if the request fails.
-   */
-  async getExercises() {
+  async getExercises(params) {
     try {
       const { data } = await this.#axios.get(
-        `${this.#basePath}?${
-          this.#filter && this.#name ? `${this.#filter}=${this.#name}&` : ''
-        }&page=${this.#page}&limit=10`
+        `${this.#basePath}?${params.toString()}`
```
Comment (snegurova) [inline src/js/api/ExercisesService.js:55]: `const { data } = await this.#axios.get(this.#basePath, params) `

File: src/js/router/router.js [hunk 3]
```diff
@@ -27,14 +28,12 @@ const routes = {
     apiCallBack: (params) => console.log(`your data and ${params}`),
   },
   '/exercises': {
-    route: `${basePath}/exercises.html`,
-    domCallBack: () => console.log('your element'),
-    apiCallBack: (params) => console.log(`your data and ${params}`),
+    route: '/pages/exercises.html',
```
Comment (snegurova) [inline src/js/router/router.js]: Please update main and be carefull
Should be
route: '${basePath}/exercises.html',

File: src/js/pages/exercises.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+export const getMainExercises = () => {
+  contentElement = document.querySelector('.content');
+};
+
+export const getExercises = async (params) => {
+  exercises = await api.exercises.getExercises(params);
+  const markup = await renderExercises(exercises);
+
+  contentElement.innerHTML = `<ul class="main-exercises">${markup}</ul>`;
+};
```
Comment (snegurova) [inline src/js/pages/exercises.js:17]: Please update main and put it to the src root

**Review by snegurova — APPROVED** [review]

File: src/js/api/ExercisesService.js (hunk 2 above)
Comment (MKuzich) [reply src/js/api/ExercisesService.js:55]: I tried this approach, but it wasn't working correctly

File: src/css/exercises.css (hunk 1 above)
Comment (MKuzich) [reply src/css/exercises.css:156]: Didn't find variables yesterday xD will change tomorrow)

File: src/js/router/router.js (hunk 3 above)
Comment (MKuzich) [reply src/js/router/router.js]: Ok

File: src/js/pages/exercises.js (hunk 4 above)
Comment (MKuzich) [reply src/js/pages/exercises.js:17]: Ok

**Pull request merged**